### PR TITLE
Make all keys in global_preferences.xml lowercase

### PR DESCRIPTION
**Pull request by RobinBossart**
Related issue #2213

### CODE REVIEW

**Comment by philipwhiuk** [comment]
We need to consider the upgrade process here.

File: k9mail/src/main/java/com/fsck/k9/K9.java [hunk 1]
```diff
@@ -467,9 +467,9 @@ public static void save(StorageEditor editor) {
 
         editor.putString("language", language);
         editor.putInt("theme", theme.ordinal());
-        editor.putInt("messageViewTheme", messageViewTheme.ordinal());
-        editor.putInt("messageComposeTheme", composerTheme.ordinal());
-        editor.putBoolean("fixedMessageViewTheme", useFixedMessageTheme);
+        editor.putInt("message_view_theme", messageViewTheme.ordinal());
+        editor.putInt("message_compose_theme", composerTheme.ordinal());
+        editor.putBoolean("fixed_message_view_theme", useFixedMessageTheme);
```
Comment (dschuermann) [inline k9mail/src/main/java/com/fsck/k9/K9.java]: I don't think changing them here is correct!

File: k9mail/src/main/java/com/fsck/k9/preferences/GlobalSettings.java [hunk 2]
```diff
@@ -348,18 +348,18 @@
...
     private static class SettingsUpgraderV24 implements SettingsUpgrader {
 
         @Override
         public Set<String> upgrade(Map<String, Object> settings) {
-            K9.Theme messageViewTheme = (K9.Theme) settings.get("messageViewTheme");
+            K9.Theme messageViewTheme = (K9.Theme) settings.get("message_view_theme");
             K9.Theme theme = (K9.Theme) settings.get("theme");
             if (theme != null && messageViewTheme != null && theme == messageViewTheme) {
-                settings.put("messageViewTheme", K9.Theme.USE_GLOBAL);
+                settings.put("message_view_theme", K9.Theme.USE_GLOBAL);
```
Comment (dschuermann) [inline k9mail/src/main/java/com/fsck/k9/preferences/GlobalSettings.java]: Same goes for all changes in GlobalSettings: I don't think changing them here is correct!

**Comment by RobinBossart** [comment]
So the only modifications needed are in prefs.java and global_preferences.xml ?

**Comment by dschuermann** [comment]
I think so yes.

**Comment by RobinBossart** [comment]
I reverted modifications in K9.java and GlobalSettings.java, please tell me if it is correct or not.

**Comment by cketti** [comment]
Looks like your reverted all changes but the ones in `K9` and `GlobalSettings`.

**Comment by RobinBossart** [comment]
Good revert is done

**Comment by cketti** [comment]
You can check the [Files changed](https://github.com/k9mail/k-9/pull/2226/files) tab so see if your changes had the intended effect.

In this case you reverted back to what you had after the first commit. Which is still wrong. We only want the changes to `Prefs.java` and `global_preferences.xml`.

**Comment by RobinBossart** [comment]
I did the revert of useless commits

**Comment by cketti** [comment]
And it only took you 7 additional commits to do so ;)

Please rewrite the history of your branch to only one commit and force push to your PR branch. See e.g. http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Comment by RobinBossart** [comment]
Commits have been squashed, tell me if everything is OK.

**Comment by cketti** [comment]
Thanks!

**Pull request merged**